tests/ParserTest.elm

```diff
@@ -605,6 +605,85 @@ expr =
                   , "\"'\""
                   , Ok (Literal (String "'"))
                   )
+                , ( "double quote"
+                  , "\"\\\"\""
+                  , Ok (Literal (String "\""))
+                  )
+                , ( "escape n"
+                  , "\"\\n\""
+                  , Ok (Literal (String "\n"))
+                  )
+                , ( "escape r"
+                  , "\"\\r\""
+                  , Ok (Literal (String "\u{000D}"))
+                  )
+                , ( "escape t"
+                  , "\"\\t\""
+                  , Ok (Literal (String "\t"))
+                  )
+                , ( "emoji"
+                  , "\"😃\""
+                  , Ok (Literal (String "😃"))
+                  )
+                , ( "escaped unicode code point"
+                  , "\"\\u{1F648}\""
+                  , Ok (Literal (String "🙈"))
+                  )
+                , ( "combo of escapes and chars"
+                  , "\"\\u{1F648}\\n\\r\\t\\abc123\""
+                  , Ok (Literal (String "🙈\n\r\tabc123"))
+                  )
+                ]
+              )
+            , ( "literal multiline string"
+              , [ ( "empty"
+                  , "\"\"\"\"\"\""
+                  , Ok (Literal (String ""))
+                  )
+                , ( "one space"
+                  , "\"\"\" \"\"\""
+                  , Ok (Literal (String " "))
+                  )
+                , ( "newline"
+                  , "\"\"\"\n\"\"\""
+                  , Ok (Literal (String "\n"))
+                  )
+                , ( "two numbers"
+                  , "\"\"\"42\"\"\""
+                  , Ok (Literal (String "42"))
+                  )
+                , ( "single quote"
+                  , "\"\"\"'\"\"\""
+                  , Ok (Literal (String "'"))
+                  )
+                , ( "double quote"
+                  , "\"\"\"\"\"\"\"\""
+                  , Ok (Literal (String "\""))
+                  )
+                , ( "escape n"
+                  , "\"\"\"\\n\"\"\""
+                  , Ok (Literal (String "\n"))
+                  )
+                , ( "escape r"
+                  , "\"\"\"\\t\"\"\""
+                  , Ok (Literal (String "\u{000D}"))
+                  )
+                , ( "escape t"
+                  , "\"\"\"\\t\"\"\""
+                  , Ok (Literal (String "\t"))
+                  )
+                , ( "emoji"
+                  , "\"\"\"😃\"\"\""
+                  , Ok (Literal (String "😃"))
+                  )
+                , ( "escaped unicode code point"
+                  , "\"\"\"\\u{1F648}\"\"\""
+                  , Ok (Literal (String "🙈"))
+                  )
+                , ( "combo of escapes, newlines, and chars"
+                  , "\"\"\"\\u{1F648}\\n\n\n\\r\\t\\abc123\"\"\""
+                  , Ok (Literal (String "🙈\n\n\n\r\tabc123"))
```

Comment:
the slashes right before `abc123` aren't in the output string ... that's probably an error?

Reply:
It is an error. Fixed by https://github.com/elm-in-elm/compiler/pull/31/commits/b74c5be275ae43548dce26d2647ff165cebc2e70, thanks